pkg/storages/swift/folder.go

```diff
@@ -151,6 +153,11 @@ func (folder *Folder) PutObject(name string, content io.Reader) error {
 	return nil
 }
 
+func (folder *Folder) PutObjectWithContext(ctx context.Context, name string, content io.Reader) error {
```

Comment:
`ObjectPut` actually [supports](https://github.com/ncw/swift/blob/b37a86bc3491c732a2b5ea198bc7cb307e239992/swift.go#L1597C4-L1597C4) passing the context, maybe just bump the swift library version and use it?